make update_docker fails with AssertionError: Multiple .dist-info directories

I ran into this with a recent code pull, possibly related to https://github.com/mozilla/addons-server/commit/de7d9f724a033e82768b2ffb5d736bfbf725dc8d

For sanity, I completely destroyed all my docker containers, pulled new ones, and ran `make initialize_docker`. Everything seems fine. I then ran `make update_docker` and I ran into this traceback:

````
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python2.7/site-packages/pip/commands/install.py", line 342, in run
    prefix=options.prefix_path,
  File "/usr/lib/python2.7/site-packages/pip/req/req_set.py", line 784, in install
    **kwargs
  File "/usr/lib/python2.7/site-packages/pip/req/req_install.py", line 851, in install
    self.move_wheel_files(self.source_dir, root=root, prefix=prefix)
  File "/usr/lib/python2.7/site-packages/pip/req/req_install.py", line 1064, in move_wheel_files
    isolated=self.isolated,
  File "/usr/lib/python2.7/site-packages/pip/wheel.py", line 345, in move_wheel_files
    clobber(source, lib_dir, True)
  File "/usr/lib/python2.7/site-packages/pip/wheel.py", line 305, in clobber
    ', '.join(info_dir))
AssertionError: Multiple .dist-info directories: /usr/lib/python2.7/site-packages/Jinja2-2.8.dist-info, /usr/lib/python2.7/site-packages/Jinja2-2.8.1.dist-info
````

Here is the [complete output of make update_docker](https://github.com/mozilla/addons-server/files/685406/docker-log.txt).

Maybe the built container doesn't have the new Jinja2 in it? Or maybe we don't need the `update_deps` step in `make update_docker` anymore? I'm not entirely sure what's happening.

--- a/requirements/docs.txt
+++ b/requirements/docs.txt
@@ -1,7 +1,7 @@
 # Jinja2 is required by Sphinx
-Jinja2==2.8 \
-    --hash=sha256:1cc03ef32b64be19e0a5b54578dd790906a34943fe9102cfdae0d4495bd536b4 \
-    --hash=sha256:bc1ff2ff88dbfacefde4ddde471d1417d3b304e8df103a7a9437d47269201bf4
+Jinja2==2.8.1 \
+    --hash=sha256:3997cf273f1424207c60d5895264f74483fce72702f15a7cd51a8551d43663ca \
+    --hash=sha256:35341f3a97b46327b3ef1eb624aadea87a535b8f50863036e085e7c426ac5891
 # Pygments is required by Sphinx
 Pygments==2.1.3 \
     --hash=sha256:485602129949b14247e8b124d28af4654dffbd076537c4a9c44a538a2c1755b7 \

